core/base/src/main/java/org/eclipse/dataspaceconnector/core/CoreServicesExtension.java

```diff
@@ -130,6 +131,9 @@ public void initialize(ServiceExtensionContext context) {
 
         var scopeFilter = new ScopeFilter(bindingRegistry);
 
+        var typeManager = context.getTypeManager();
+        RegistrationTypes.getRegistrationTypes().forEach(typeManager::registerTypes);
```

Comment:
Could this be renamed to `PolicyRegistrationTypes`? Also, why not use the `TYPES` constant directly? Would make it obvious that it is in fact a constant.

Reply:
Yep. That has been in there for a while (i.e. forever). Let's come back to that in another PR.

Reply:
#946